src/ucp/core/ucp_request.inl

```diff
@@ -64,12 +64,15 @@
 
 #define ucp_request_complete(_req, _cb, _status, ...) \
     { \
+        /* NOTE: we have to store "RELEASED" flag here to provide backward */ \
```

Comment:
what does this mean?

Reply:
ucp_request_release function doesn't cancel completion call, to support such behavior we have to store "RELEASED" flag here

Reply:
updated comment